doc/nrf/device_guides/working_with_nrf/nrf53/nrf5340.rst

```diff
@@ -579,6 +579,57 @@ Bluetooth buffers configuration introduced by the :kconfig:option:`CONFIG_NCS_SA
+This configuration typically follows:
+
++-------+-------------------------------------------------------------+-----------+-------+
+| Slot  | Description                                                 | DTS label | n     |
++=======+=============================================================+===========+=======+
+| slot0 | First image's primary slot (internal flash)                 | "image-0" | 1     |
++-------+-------------------------------------------------------------+-----------+-------+
+| slot1 | First image's secondary slot (internal or external flash)   | "image-1" | 0, 2  |
++-------+-------------------------------------------------------------+-----------+-------+
+| slot2 | Second image's primary slot (internal RAM)                  | "image-2" | 3     |
++-------+-------------------------------------------------------------+-----------+-------+
+| slot3 | Second image's secondary slot (internal or external flash)  | "image-3" | 4     |
++-------+-------------------------------------------------------------+-----------+-------+
+
+.. note::
+   The default upload target for firmware updates using the **mcumgr-cli** tool and the ``CONFIG_MCUBOOT_SERIAL_DIRECT_IMAGE_UPLOAD`` Kconfig option enabled is the secondary slot of the first image, denoted as slot1.
+   In this context, the default behavior without specifying a slot number corresponds to ``-n 0``.
+
+   For non-default behavior targeting the secondary slot of an image other than the first, use ``-n 2``.
+   This adjustment is due to the default slot numbering and the shift in identifiers when a non-default upload to the secondary slot is selected.
+
+For the nRF5340 DK, serial recovery differs based on the operation mode:
+
+* In the case of serial recovery:
+
+  * For updates to the network core, specify ``-n 3``.
+    This indicates that updates are mediated through the application core's RAM, a process specific to the nRF5340 DK in its default configuration.
+
+* In contrast, the behavior differs when uploading images through different methods:
+
+  * For Device Firmware Updates (DFU):
+
+    * If no image number is provided or ``-n 0`` is used, the default behavior is to upload the firmware to slot1, which is the secondary slot of the first image.
+
+  * For MCUboot:
+
+    * The primary slot of the first image (slot0) does not require a specific image number to be targeted for updates.
+
+This distinction is important for ensuring the correct firmware update process is followed.
+The terms "slot" and "image" are used distinctly; "slot" refers to the physical or logical memory locations, while "image-x" refers to labels historically used in devicetree (DTS) configurations, which correspond to specific slots.
+Note that there are two slots per application image: primary and secondary, with primary slots being those with odd numbers and secondary slots being those with even numbers.
```

Comment:
This was info for you sot that we were on the same page regarding what I try to communicate. I do not think it is worth adding to documentation.